amenity-points.mss

```diff
@@ -27,8 +27,20 @@
     point-placement: interior;
   }
 
+  [feature = 'tourism_wilderness_hut'][zoom >= 13],
+  [feature = 'amenity_shelter']['shelter_type' = 'basic_hut'][zoom >= 14],
   [feature = 'amenity_shelter'][zoom >= 16] {
     marker-file: url('symbols/shelter-14.svg');
+    [feature = 'tourism_wilderness_hut'] {
+    	marker-file: url('symbols/wilderness_hut.svg');
+    	marker-transform:"scale(0.6)";
```

Comment:
The marker should be sized an appropriate size to not need a transform. Also, the same SVG will not look good at two different scales.